tendrl-commons.spec

```diff
@@ -18,7 +18,7 @@ Requires: python-maps
 Requires: python-dateutil
 Requires: python-etcd
 Requires: python-six
-Requires: python-ruamel-yaml
+Requires: python2-ruamel-yaml
```

Comment:
Is this available as part of epel or we build it?

Reply:
yes its available in epel

Reply:
Ack. Looks fine otherwise